rust/nft-candy-machine/src/lib.rs

```diff
@@ -26,7 +26,21 @@ pub mod nft_candy_machine {
     };
 
     use super::*;
+    pub fn withdraw_funds<'info>(
+        ctx: Context<'_, '_, '_, 'info, WithdrawFunds<'info>>,
+    ) -> ProgramResult {
+        let authority = &ctx.remaining_accounts[0];
```

Comment:
Now this won't work anymore since you aren't sending up remaining accounts

Reply:
I may? have done my git commit in the wrong dir?

Reply:
shoudl be ctx.accounts.authority, because now you aren't sending up remaining accounts anymore

Reply:
why is my code not doing what it's supposed to do

Reply:
![image](https://user-images.githubusercontent.com/22843601/139601045-9fd43ea6-fbcc-4197-b458-e294b6e07849.png)

I swear I did this

Reply:
oh, yeah, @bhgames check the newest push

I had indeed git commit in a subdir of js